ui/src/store/category.ts

```diff
@@ -0,0 +1,95 @@
+import { types, getEnv, flow, Instance } from "mobx-state-tree";
+import { Api } from "../api";
+
+const Tag = types.model("Tags", {
+	id: types.integer,
+	name: types.string,
+});
+
+export const Category = types
+	.model("Category", {
+		id: types.number,
```

Comment:
`types.number`  & `types.integer` ? which would be better to use? I guess `types.number` can hold float value also.

Reply:
While comparing the id while toggling the selected field of category we use id, but typescript don't have integer type whereas mst has both integer and number hence used number as type